<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="library">
         <property name="styleSheet">
          <string notr="true">QLabel#library {
	font-size: 50px;
	font-family: 微软雅黑;
}</string>
         </property>
         <property name="text">
          <string>图书馆</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="tuijian_button">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>推荐图书</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="book_button">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>书城</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="collect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#collect {
	background-color: rgb(173, 216, 230);
}</string>
         </property>
         <property name="text">
          <string>收藏图书</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reserve">
         <property name="text">
          <string>预约图书</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="report_loss">
         <property name="text">
          <string>挂失</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="personal_information">
         <property name="text">
          <string>个人信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>收藏</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">QScrollArea#scrollArea {
	border: none;
}</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>642</width>
           <height>237</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
